tt_metal/impl/device/device_pool.cpp

```diff
@@ -356,6 +356,9 @@ std::vector<Device*> DevicePool::get_all_active_devices() const {
 }
 
 bool DevicePool::close_device(chip_id_t device_id) {
+    // Sync and close one device
+    // Currently can only call this on mmio chips, once we split dispatch kernel shutdown
+    // from device close, we can call this on remote devices too
     tt::Cluster::instance().set_internal_routing_info_for_ethernet_cores(false);
     bool pass = true;
     const auto& mmio_device_id = tt::Cluster::instance().get_associated_mmio_device(device_id);
```

Comment:
it feels weird to me that calling `DevicePool::close_device` on non-mmio chip triggers it instead on the mmio_chip. Can't we now do this:
```
tt::Cluster::instance().set_internal_routing_info_for_ethernet_cores(false, {device_id});
```